app.js

```diff
@@ -1,15 +1,17 @@
 const express = require("express");
 const app = express();
 const { getTopics } = require("./controllers/controller");
-
+const endpoints = require("./endpoints.json");
 app.get("/api/topics", getTopics);
 
+app.get("/api", (request, response) => {
+  response.status(200).send({ endpoints });
```

Comment:
As responses are usually sent from the controller, I would advise sending the response from a function there 🙂